src/main/resources/Office365/Office365Macro.xml

```diff
@@ -568,6 +568,13 @@ if (url!=null &amp;&amp; url!="") {
    doc.set("fileName", request.filename)
    doc.set("user", xcontext.user)
    doc.save("Inserting Office365 Document")
+   def path = request.getPathInfo();
+   if(path!=null &amp;&amp; path.length()>0) {
```

Comment:
Same remarks as above, for this block code.

Reply:
Fixed in https://github.com/xwikisas/application-office365/pull/11/commits/73ead5a9b328d8ac9d7dd47a720688076251f346 . thanks to check.